channels/uk.m3u

```diff
@@ -391,6 +433,18 @@ https://qwestclassic-samsung-uk.amagi.tv/playlist.m3u8
 https://qwestjazz-samsung-uk.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="QUESTTVMIX.uk" tvg-name="QUEST TV MIX" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/B69BHnx.jpg" group-title="",QUEST TV MIX (720p)
 https://qwestmix-samsung-uk.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="QVC.uk" tvg-name="QVC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/q8jkrYW.png" group-title="Shop",QVC
+https://d1txbbj1u9asam.cloudfront.net/live/qvcuk_main_clean/bitrate1.isml/3/prog_index.m3u8
+#EXTINF:-1 tvg-id="QVC.uk" tvg-name="QVC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/q8jkrYW.png" group-title="Shop",QVC +1
+https://live-qvcuk.simplestreamcdn.com/hera/remote/qvcuk_primary_sdi1/live.m3u8
+#EXTINF:-1 tvg-id="QVCBeauty.uk" tvg-name="QVC Beauty" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/Xb2t2EF.jpg" group-title="Shop",QVC Beauty
+http://live.qvcuk.simplestreamcdn.com/live/qvcuk_beauty_clean/bitrate1.isml/live.m3u8
```

Comment:
Originates from the US.

Reply:
https://www.qvcuk.com/content/shop-live-tv.html

Reply:
Accepting it seems QVC had made an arrangement with Sky to create their own QVC, and is streamed from the UK : https://gaz.wiki/wiki/fi/QVC_(British_TV_channel)

Reply:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.